docs/content/en/docs/tutorials/tetragon-metrics.md

```diff
@@ -73,6 +73,40 @@ promhttp_metric_handler_errors_total{cause="gathering"} 0
 ...
 ```
 
-### Container
+## Container
 
-TBD
\ No newline at end of file
+Tetragon, when installed as a container as mentioned in [Container Deployment]({{<ref "docs/getting-started/deployment/container" >}})
+
+## Package
+
+Tetragon, when installed via release packages as mentioned in [Package Deployment](../getting-started/deployment/package.md). It can be interacted with `tetragon` command. By default, metrics are disabled, which can be enabled using `--metrics-server` flag, by specifying the address.
+
```

Comment:
Hi @prateek041 please add that you can set the metrics address server by creating a file under `/etc/tetragon/tetragon.conf.d/metrics-server` with the address server as content, example `localhost:2112`

Reply:
also please reference https://tetragon.cilium.io/docs/reference/tetragon-configuration/#configuration-examples for further examples

Reply:
looking into it.

Reply:
I added a [section](https://github.com/cilium/tetragon/pull/1495/files#diff-575dbe46cb3476f4aa4e207f5d24d5eaca95ec56bb691106be85b3c5e87f42b0R45) does this work ?

@tixxdz